graphql_compiler/cost_estimation/filter_selectivity_utils.py

```diff
@@ -337,6 +339,47 @@ def _combine_filter_selectivities(selectivities):
     return Selectivity(kind=combined_selectivity_kind, value=combined_selectivity_value)
 
 
+def _is_int_field_type(schema_info, location_name, field_name):
+    field_type = schema_info.schema.get_type(location_name).fields[field_name].type
+    return field_type == GraphQLInt
+
+
+def _get_selectivity_fraction_of_interval(interval, quantiles):
+    """Get the fraction of values contained in an interval.
+
+    We ignore the interval endpoint values, and only consider the quantile they
+    are in. We treat the endpoints as uniform random variables within their quantile
+    and compute the expected value of the length of the interval divided by the size
+    of the whole domain.
+
+    Args:
+        interval: IntegerInterval defining the range of values
+        quantiles: Sorted list of N quantiles dividing the values into N+1 groups of
+                   almost equal size.
+
+    Returns:
+        float, the fraction of the values contained in the interval.
+    """
+    domain_interval_size = len(quantiles) + 1
+    if interval.lower_bound is None and interval.upper_bound is None:
+        interval_size = domain_interval_size
+    elif interval.lower_bound is None:
+        upper_bound_quantile = bisect.bisect_left(quantiles, interval.upper_bound)
+        interval_size = 0.5 + upper_bound_quantile
+    elif interval.upper_bound is None:
+        lower_bound_quantile = bisect.bisect_left(quantiles, interval.lower_bound)
+        interval_size = len(quantiles) - lower_bound_quantile
+    else:
+        lower_bound_quantile = bisect.bisect_left(quantiles, interval.lower_bound)
+        upper_bound_quantile = bisect.bisect_left(quantiles, interval.upper_bound)
+        if lower_bound_quantile == upper_bound_quantile:
+            # https://math.stackexchange.com/questions/195245/
+            interval_size = 1.0 / 3
```

Comment:
Can you make this a constant and put that link above? That way we don't need to include that comment in multiple places.

Reply:
I think that's less readable. I don't have to create a whole concept about this, give it a name, and a special place. It's not an important piece of this code and it does not deserve more attention.